contracts/contracts/coordination/subscription/BqETHSubscription.sol

```diff
@@ -21,17 +21,19 @@ contract BqETHSubscription is EncryptorSlotsSubscription, Initializable, Ownable
         uint128 encryptorSlots; // pre-paid encryptor slots for the billing period
     }
 
-    IERC20 public immutable feeToken;
-
     uint32 public constant INACTIVE_RITUAL_ID = type(uint32).max;
+    uint192 public constant INCREASE_BASE = 10000;
 
+    GlobalAllowList public immutable accessController;
+    IERC20 public immutable feeToken;
     address public immutable adopter;
 
-    uint256 public immutable baseFeeRate;
+    uint256 public immutable initialBaseFeeRate;
     uint256 public immutable encryptorFeeRate;
     uint256 public immutable maxNodes;
 
-    GlobalAllowList public immutable accessController;
+    uint256 public immutable baseFeeRateIncrease;
```

Comment:
just nitpicking, but can we place this a few lines above, together with the other fee rate stuff?

Reply:
✔️